### PR TITLE
fix(android): Fixed bug where Label "ellipsize" modes START/MIDDLE don't work if "autoLink" is true

### PR DESCRIPTION
**Pull request by jquick-axway**
**JIRA:**
https://jira.appcelerator.org/browse/TIMOB-24881

**Summary:**
- This was a limitation in Google's `LinkMovementMethod` Java class which only supported an `END` ellipsize mode.
- Google's `TextView` Java class on Android 4.4 and above supports links without a `MovementMethod`. So, the solution is to remove its usage since Titanium 8.0.0 min Android version supported is 4.4.

**Test:**
_(The below test **MUST** be ran on Android 4.4 as well.)_
1. Build and run the below code on Android.
2. Verify that ellipsis `...` are shown at the beginning of the text link.
3. Tap on the link and verify that it takes you to: https://www.appcelerator.com
4. Comment out the below code's `Ti.UI.TEXT_ELLIPSIZE_TRUNCATE_START` line.
5. Uncomment the below code's `Ti.UI.TEXT_ELLIPSIZE_TRUNCATE_MIDDLE` line.
6. Build and run on Android.
7. Verify that ellipsis `...` appear in the middle of the text link.
8. Tap on the link and verify that it takes you to: https://www.appcelerator.com
9. Comment out the below code's `Ti.UI.TEXT_ELLIPSIZE_TRUNCATE_MIDDLE` line.
10. Uncomment the below code's `Ti.UI.TEXT_ELLIPSIZE_TRUNCATE_END` line.
11. Build and run on Android.
12. verify that ellipsis `...` appear at the end of the text link.
13. Tap on the link and verify that it takes you to: https://www.appcelerator.com

```javascript
var window = Ti.UI.createWindow();
window.add(Ti.UI.createLabel({
	text: "https://www.appcelerator.com",
	font: { fontSize: "50dp" },
	autoLink: Ti.UI.AUTOLINK_URLS,
	ellipsize: Ti.UI.TEXT_ELLIPSIZE_TRUNCATE_START,
//	ellipsize: Ti.UI.TEXT_ELLIPSIZE_TRUNCATE_MIDDLE,
//	ellipsize: Ti.UI.TEXT_ELLIPSIZE_TRUNCATE_END,
	maxLines: 1,
	width: "100%",
}));
window.open();
```

### CODE REVIEW

File: android/modules/ui/src/java/ti/modules/titanium/ui/widget/TiUILabel.java [hunk 1]
```diff
@@ -26,8 +26,6 @@
 import android.text.Html;
 import android.text.InputType;
```
Comment (ypbnv) [inline android/modules/ui/src/java/ti/modules/titanium/ui/widget/TiUILabel.java]: Not related to this PR, but would you be able to remove the following imports as we are on this file:
`import android.text.InputType;`
`import android.os.Build;`
They don't seem to be used in the file anymore.

**Review by ypbnv — APPROVED** [review]
CR: Pass

File: android/modules/ui/src/java/ti/modules/titanium/ui/widget/TiUILabel.java (hunk 1 above)
Comment (jquick-axway) [reply android/modules/ui/src/java/ti/modules/titanium/ui/widget/TiUILabel.java]: Thanks for catching that. I'll remove them.

**Review by ssjsamir — APPROVED** [review]
**FR Passed** Label "ellipsize" modes START/MIDDLE now work if "autoLink" is true, tested using the instructions mentioned above. 

**Test Environment**
```
Nexus 5X 4.4.2
Google pixel xl 7.1.1 sim
7.1.0-master.16
Operating System Name: Mac OS Mojave
Operating System Version: 10.14.2
Node.js Version: 8.11.3
```

**Pull request merged**